src/main/java/com/dawn/jat/illuminati/post/controller/PostController.java

```diff
@@ -62,23 +64,26 @@
      * Create a Post in to database.
      */
     @PostMapping(value = "/create")
-    public ResponseEntity<Object> createPost(@RequestBody PostDto post) {
-        PostEntity savedPost = postService.create(post);
-        SuccessResponse resp = new SuccessResponse(savedPost);
+    public ResponseEntity<?> createPost(@Validated @RequestBody PostDto post) {
```

Comment:
for `@Validated` We dont have any Validated attribute in `PostDto`, should we add some validated attribute to `PostDto` if we want to enable this?
Detail Usage for steps: https://blog.codeleak.pl/2014/08/validation-groups-in-spring-mvc.html

Reply:
Removed @Validated, will consider later when needed